scripts/histmakers/mw_with_mu_eta_pt.py

```diff
@@ -54,21 +54,15 @@
             logger.warning("Running theory agnostic with only nominal and mass weight histograms for now.")
             parser = common.set_parser_default(parser, "onlyMainHistograms", True)
     if args.unfolding:
+        #TODO: Fix
         parser = common.set_parser_default(parser, "pt", [32,26.,58.])
 
-# axes for W MC efficiencies with uT dependence for iso and trigger
```

Comment:
Similarly I would just introduce different axes (ptcourse, etacourse for example) and just run the analysis with those options. I kept the modification of the selection needed.

Reply:
As I wrote for the part about the fakes, the complication is that in some cases you also need a custom selection. In principle this might deserve a completely new histmaker, since most of the main one is unnecessary, but as usual this implies some code duplication and potential desynchronisation between the two scripts. In this case we may consider it, since the W MC efficiencies are obtained running without scale factors and most of other experimental corrections, but we probably still want to keep consistent theory corrections applied at gen level